code/hdb/hdbstandard.q

```diff
@@ -4,4 +4,28 @@ reload:{
 	system"l ."}
 
 // Get the relevant HDB attributes
-.proc.getattributes:{`date`tables!(@[value;`date;`date$()];tables[])}
+\d .proc
+getattributes:{
+	timecolumns:1!except[(::),gettimecolumns each t:tables[`.];(::)];
```

Comment:
why is the generic null stuff required here? (assuming it is necessary, could do with a comment above to clarify reason I think)